pydcs_extensions/mig31bm/mig31bm.py

```diff
@@ -0,0 +1,98 @@
+
+from typing import Any, Dict, Set
+
+from dcs import task
+from dcs.planes import PlaneType
+from dcs.weapons_data import Weapons
+
+from game.modsupport import planemod
+from pydcs_extensions.weapon_injector import inject_weapons
+
+class WeaponsMiG31BM:
+
+    Vympel_R_33__AA_9_Amos_ = {"clsid": "{R-33}", "name": "Vympel R-33 (AA-9 Amos)", "weight": 490}
+    Vympel_R_37M__AA_13_Axehead_ = {"clsid": "{R-37M}", "name": "Vympel R-37M (AA-13 Axehead)", "weight": 600}
+    Vympel_R_37__AA_13_Axehead_ = {"clsid": "{R-37}", "name": "Vympel R-37 (AA-13 Axehead)", "weight": 600}
+
+inject_weapons(WeaponsMiG31BM)
+
+@planemod
+class MiG_31BM(PlaneType):
+    id = "MiG-31BM"
+    flyable = True
+    height = 5.15
+    width = 13.64
+    length = 22.69
+    fuel_max = 17730
+    max_speed = 3120.012
+    chaff = 120
+    flare = 100
+    charge_total = 220
+    chaff_charge_size = 1
+    flare_charge_size = 1
+    category = "Interceptor"  #{78EFB7A2-FD52-4b57-A6A6-3BF0E1D6555F}
+
+    livery_name = "MIG-31BM"  # from type
+
+    class Pylon1:
+        R_73__AA_11_Archer____Infra_Red = (1, Weapons.R_73__AA_11_Archer____Infra_Red)
+        R_77__AA_12_Adder____Active_Rdr = (1, Weapons.R_77__AA_12_Adder____Active_Rdr)
+
+    class Pylon2:
+        R_73__AA_11_Archer____Infra_Red = (2, Weapons.R_73__AA_11_Archer____Infra_Red)
+        R_77__AA_12_Adder____Active_Rdr = (2, Weapons.R_77__AA_12_Adder____Active_Rdr)
+        Vympel_R_37M__AA_13_Axehead_ = (2, Weapons.Vympel_R_37M__AA_13_Axehead_)
+        Vympel_R_37__AA_13_Axehead_ = (2, Weapons.Vympel_R_37__AA_13_Axehead_)
+        Vympel_R_33__AA_9_Amos_ = (2, Weapons.Vympel_R_33__AA_9_Amos_)

```

Comment:
These are custom weapons, so the need to reference the WeaponsMiG31BM class instead.